src/config/dynamic-theme-fixes.config

```
@@ -951,6 +951,15 @@ CSS
 
 ================================
 
+apnews.com
+
+INVERT
+.hamburger-box
+.sticky-part > div > * > svg
+.LeftRail > div > * > svg
```

```suggestion
.sticky-part > div svg
.LeftRail > div svg
```

I don't like to target `> *`